lib/tables.js

```diff
@@ -47,7 +47,15 @@ exports.STDLIB_MATH_TYPES = dict({
     'LOG10E':  ty.Double,
     'PI':      ty.Double,
     'SQRT1_2': ty.Double,
-    'SQRT2':   ty.Double
+    'SQRT2':   ty.Double,
+    'min':     ty.Overloaded([
+                   ty.Arrow([ty.Int, ty.Int], ty.Signed),
+                   ty.Arrow([ty.Double, ty.Double], ty.Double)
+               ]),
+    'max':     ty.Overloaded([
+                   ty.Arrow([ty.Int, ty.Int], ty.Signed),
+                   ty.Arrow([ty.Double, ty.Double], ty.Double)
+               ])
```

Comment:
For tidiness, please put min and max after imul, so that all the functions remain grouped together.

FWIW, in the spec, Math.min and Math.max are variadic, though supporting 2-input non-variadic forms is a strict improvement over nothing at all, so I wouldn't hold up the patch for that.